intake/cli/server/server.py

```diff
@@ -84,13 +85,25 @@ def initialize(self, cache, catalog, auth):
 
     def get(self):
         head = self.request.headers
+        page_size = self.get_argument('page[size]', None)
```

Comment:
was wondering why you use `page[size]` as opposed to `page_size` or the object `page: {size:, number/offset"}`